lib/entities.js

```diff
@@ -2160,7 +2161,8 @@ exports.HTML_ENTITIES = freeze({
 });
 
 /**
- * @deprecated use `HTML_ENTITIES` instead
- * @see HTML_ENTITIES
+ * @see HTML_ENTITIES.
+ * @deprecated
+ * Use `HTML_ENTITIES` instead.
```

Comment:
I really want `@deprecated` to always be the first tag to be listed if it is provided, so we should configure the tag order.

Reply:
There is [`jsdocTagsOrder`](https://github.com/homer0/packages/blob/main/packages/public/prettier-plugin-jsdoc/README.md#tags-order)